kong/db/declarative/init.lua

```diff
@@ -660,6 +660,11 @@ end
 declarative.unique_field_key = unique_field_key
 
 
+local function config_is_empty(entities)
+  -- empty configuration has no entries other than workspaces
+  return entities["workspaces"] and tablex.size(entities) == 1
```

Comment:
You can use https://github.com/openresty/luajit2#tablenkeys for `tablex.size`. Might be faster for large configs.

Also, do we need to do `and entities.workspace` just to be safe? I'm not sure if it is possible to have an `entities` that has not workspace or will it always be added by the parser.